model/cookies-preference.go

```diff
@@ -1,9 +1,10 @@
 package model
 
-import "github.com/ONSdigital/dp-renderer/model"
+import "github.com/ONSdigital/dp-renderer/v2/model"
 
 // CookiesPreference is the model struct for the cookies preferences form
 type CookiesPreference struct {
 	model.Page
-	PreferencesUpdated bool `json:"preferences_updated"`
+	PreferencesUpdated bool                `json:"preferences_updated"`
+	UsageRadios        model.RadioFieldset `json:"type_radios"`
```

Comment:
The json needs to be updated with the new name
```suggestion
	UsageRadios        model.RadioFieldset `json:"usage_radios"`
```